apps/website/versioned_docs/version-v2.x/use-cases/public-goods.md

```diff
@@ -1,54 +1,40 @@
 
-### Markets
+There are different flavors of funding mechanisms, you can find more information about each of them below:
 
-Markets can be thought of as fund-distributors: sellers are the fund recipients, and buyers are the voters deciding what will be funded. In this case, the "voting system" is 1-dollar-1-vote: each buyer has exactly as much potential influence over where funds go as they have money to spend. This is also the case for traditional crowdfunding platforms like Kickstarter.
+- [Simple Voting Funding](/use-cases/public-goods-funding/quadratic-funding/what-is-quadratic-funding.md)
+- [Quadratic Funding](/use-cases/public-goods-funding/quadratic-funding/what-is-quadratic-funding.md)
+- [Retroactive Public Goods Funding](/use-cases/public-goods-funding/retroactive-public-goods-funding/what-is-retroactive-public-goods-funding.md)
 
-An advantage of this system is that each voter's influence is directly proportional to their need (at least, if we ignore wealth inequality). The more Alice needs pizza, the more pizza she'll buy, and the more funding will go to pizzerias.
+## Simple Majority Funding
 
-The disadvantage is that rich voters have an outsize influence on what gets funded. Alice may need pizza exactly as much as Bob needs movie tickets, but if Bob is twice as wealthy as Alice, he has twice as many "votes" to spend on going to the movies.
+Public goods funding through a Simple Majority mechanism involves a pool of funds that is allocated based on the outcomes of a straightforward voting process. The percentage of votes received determines the corresponding percentage of the pool distributed.
 
-### Private Grant-Givers
+## Quadratic Funding
 
-Privately-owned grant-giving institutions, like non-profit foundations, are quite common, but they don't have voting systems. Instead, grants committees and the like solicit grant proposals and then choose which applicants will receive funding internally.
-
-An advantage of non-profits is that their non-profit status binds them to **act in the interest of some public mission**, rather than seeking profit above all.
+Quadratic funding (QF) is a crowdfunding mechanism that aims to distribute funds in a way that is more fair and inclusive than traditional crowdfunding. It works by matching small individual contributions with matching funds from a larger pool, using a quadratic formula.
 
-On the other hand, the small, private interests that control these organizations lack the tools and incentives to effectively serve large publics.
+In a QF round, donors are incentivized to contribute small amounts of money to a project or cause that they support. The matching funds are then distributed to the projects that receive the most support from the largest number of people. This helps to ensure that smaller, grassroots projects can receive funding and support, even if they don't have access to large donors or corporate sponsors.
```

Comment:
why are they incentivised to contribute small amounts? (add some explanation)

Reply:
Upon reflection, QV allows users to express a strong opinion about a cause they are passionate about. In theory, this encourages them to donate more rather than less, ultimately eliminating small donation amounts.